packages/dappmanager/src/modules/installer/https.ts

```diff
@@ -0,0 +1,166 @@
+import { listPackageNoThrow } from "../docker/list/listPackages";
+import { httpsPortal } from "../../calls/httpsPortal";
+import { prettyDnpName } from "../../utils/format";
+import params from "../../params";
+import { InstallPackageData } from "../../types";
+import { Log } from "../../utils/logUi";
+import { HttpsPortalMapping } from "../../common";
+import { getExternalNetworkAliasFromPackage } from "../../domains";
+import { ComposeFileEditor } from "../compose/editor";
+import { dockerListNetworks, dockerCreateNetwork } from "../docker";
+
+/**
+ * Recreate HTTPs portal mapping if installing or updating HTTPs package
+ */
+export async function httpsEnsureNetworkExists(
+  pkg: InstallPackageData,
```

Comment:
No need to pass `pkg` if you assert that it's the HTTPs below. Just use hardcoded values